src/test/java/com/wikia/webdriver/testcases/mobilewikitests/ArticlePageTests.java

```diff
@@ -133,14 +134,14 @@ public void mercury_article_navigateToArticlesWithColonAndQuestionMark() {
     Log.warning("Info", "Accessing article through link in navigation side");
 
     topBar.openNavigation();
-    navigation.openSubMenu(3);
+    //navigation.openSubMenu(3);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)